homeassistant/components/group/__init__.py

```diff
@@ -280,6 +287,11 @@ async def groups_service_handler(service):
         if service.service == SERVICE_SET:
             need_update = False
 
+            if ATTR_REMOVE_ENTITIES in service.data:
+                delta = service.data[ATTR_REMOVE_ENTITIES]
+                entity_ids = set(group.tracking) - set(delta)
+                await group.async_update_tracked_entity_ids(entity_ids)
```

Comment:
We shouldn't call `group.async_update_tracked_entity_ids` twice if both add and remove are passed in.